.changeset/neat-penguins-fix.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-notifications-backend': patch
+'@backstage/plugin-signals-backend': patch
+'@backstage/plugin-signals-node': patch
+---
+
+Renamed `SignalService` to `SignalsService`
```

Comment:
Prolly highlight the `signalsServiceRef` change too.

Btw, best also have a separate changeset for the backend packages, that's just internal updates due to the `signals-node` changes right?